### PR TITLE
Documentation WIP

### PR DESCRIPTION
**Pull request by loostro**
Working on documentation.

### CODE REVIEW

File: Resources/doc/cookbook/commandLine.md [hunk 1]
```diff
@@ -5,33 +5,25 @@
 
 [back-to-index]: https://github.com/symfony2admingenerator/AdmingeneratorGeneratorBundle/blob/master/Resources/doc/documentation.md#7-cookbook
 
-There are two ways to create an admin using console.
+This bundle sole purpose is to speed up the process of generateing a fully featured backend for your models.
```
Comment (javiereguiluz) [inline Resources/doc/cookbook/commandLine.md]: typo: `generateing` -> `generating`

File: Resources/doc/cookbook/commandLine.md [hunk 2]
```diff
@@ -5,33 +5,25 @@
...
+To get you start, use one of the following commands, depending on wheather your model already exists:
```
Comment (javiereguiluz) [inline Resources/doc/cookbook/commandLine.md]: Instead of `To get you start` I think it's said `To get you started`

Comment (javiereguiluz) [inline Resources/doc/cookbook/commandLine.md]: minor typo: `wheather` -> `whether`

**Comment by loostro** [comment]
@javiereguiluz thanks, fixed

**Pull request merged**